kuma/users/tests/test_templates.py

```diff
@@ -409,3 +411,263 @@ def test_common_reasons_empty(self):
         eq_(len(reasons_to_ban_found), len(reasons_to_ban_expected))
         for reason in reasons_to_ban_found:
             ok_(reason.text in reasons_to_ban_expected)
+
+
+@pytest.mark.bans
+class BanAndCleanupTestCase(UserTestCase):
+    def test_user_revisions_in_one_click_page_template(self):
+        """The user's revisions show up in the ban and cleanup template."""
+        testuser = self.user_model.objects.get(username='testuser')
+        admin = self.user_model.objects.get(username='admin')
+
+        # Create an original revision on a document by the admin user
+        document = create_document(save=True)
+        original_revision = create_revision(
+            title='Revision 0',
+            document=document,
+            creator=admin,
+            save=True)
+
```

Comment:
This "create a doc with an admin revision" code appears multiple times in two test case classes. Refactor to a shared method or ``setUp``?